app/views/govuk_publishing_components/components/_title.html.erb

```diff
@@ -13,10 +14,11 @@
   classes = %w(gem-c-title govuk-!-margin-top-8)
   classes << "gem-c-title--inverse" if inverse
   classes << (shared_helper.get_margin_bottom)
+
 %>
 <%= content_tag(:div, class: classes) do %>
   <% if context %>
-    <p class="gem-c-title__context">
+    <p class="gem-c-title__context" lang='<%= context_locale %>'>
```

Comment:
```suggestion
    <p class="gem-c-title__context" <%= "lang=#{context_locale}" if context_locale.present? %>>
```

should only display `lang` if the context_locale is present.

Reply:
Fixed